lib/vector-common/src/conversion.rs

```diff
@@ -141,6 +141,7 @@ impl Conversion {
     /// # Errors
     ///
     /// Returns errors from the underlying conversion functions. See `enum Error`.
+    #[allow(clippy::trait_duplication_in_bounds)] // appears to be a false positive
```

Comment:
Asking in Slack about this one before I create a clippy issue.

Reply:
Planning on merging regardless?